index.html

```diff
@@ -102,130 +108,151 @@ <h3 class="work-project-header">Multi-Post Stories</h3>
               ever since the 1500s, when an unknown printer took a standard
               dummy text.
             </p>
+
+            <ul class="lang-list-ul">
+              <li class="lang-list">css</li>
+              <li class="lang-list">html</li>
+              <li class="lang-list">bootstrap</li>
+              <li class="lang-list">Ruby</li>
+            </ul>
+
+            <button type="button" class="project-btn">See Project</button>
+
           </div>
-          <ul class="lang-list-ul">
-            <li class="lang-list">css</li>
-            <li class="lang-list">html</li>
-            <li class="lang-list">bootstrap</li>
-            <li class="lang-list">Ruby</li>
-          </ul>
-          <button type="button" class="project-btn">See Project</button>
         </div>
-      </div>
+      <div class="work-grid-container">
       <!--grid elements-->
       <!--grid elements 1-->
-      <div class="grid-project-container grid-1">
-        <h4 class="grid-project-header">Profesional Art Printing Data</h4>
-        <p class="grid-project-description">
+        <div class="grid-project-container grid-1">
+          <div class="grid-project1">
+            <h4 class="grid-project-header">Profesional Art Printing Data</h4>
+            <p class="grid-project-description">
```

Comment:
- [ ] In the first project is different from than others. Please change it. Also after change make sure buttons include `hover, active, press` styles.
### ORIGINAL DESKTOP
![image](https://user-images.githubusercontent.com/38635158/144269227-0fd6209c-8a58-408a-b1be-5bfe0ddff844.png)
___

- [ ] In the mobile version `see project` buttons do not exist on your page. I checked the main branch buttons exist. Please fix it. This button should take you to the Live Demo of the project or to the GitHub repository.
### YOUR DESIGN MOBİLE VERSİON
![image](https://user-images.githubusercontent.com/38635158/144269390-5acd04ea-6e74-4476-8ab2-f171190a8429.png)

Reply:
**In the desktop version, the first tile is used to show the hover effects, it is not different from other tiles.**